apps/builder/app/builder/features/sidebar-left/panels/components/components.tsx

```diff
@@ -134,6 +138,7 @@ export const TabContent = ({ publish, onSetActiveTab }: TabContentProps) => {
           dragItem: utils.tree.createInstance({ component: componentName }),
         },
       });
+      toggleCanvasPointerEvents(false);
```

Comment:
Here every change of store will trigger additional rerender because you invoke hook with subscription to store. You can just use store.set() directly to avoid this.